bindings/python/manifold3d.cpp

```diff
@@ -238,7 +238,33 @@ PYBIND11_MODULE(manifold3d, m) {
           "\n"
           ":param n: The number of pieces to split every edge into. Must be > "
           "1.")
-      .def("to_mesh", &Manifold::GetMesh)
+      .def(
+          "to_meshgl",
```

Comment:
I was thinking of leaving this as just "mesh", like we do in the JS bindings, since there's only the one kind here.